chainerx_cc/chainerx/python/routines.cc

```diff
@@ -371,6 +371,106 @@ void InitChainerxLogic(pybind11::module& m) {
           py::arg("keepdims") = false);
 }
 
+auto _split(auto &split_sections, auto &split_indices, const ArrayBodyPtr& ary, py::handle indices_or_sections, int8_t axis) {
```

Comment:
How about writing return type explicitly?
```suggestion
std::vector<ArrayBodyPtr> _split(auto &split_sections, auto &split_indices, const ArrayBodyPtr& ary, py::handle indices_or_sections, int8_t axis) {
```

Reply:
Could you fix indentation? (It seems to use 5 spaces for indentation...